lib/src/core/renovation.dart

```diff
@@ -124,18 +128,33 @@ class Renovation {
         'Accept': 'application/json',
         'X-Requested-With': 'XMLHttpRequest'
       };
+
+      frappe = Frappe(config);
+      unawaited(frappe.loadAppVersions());
```

Comment:
@assemmarwan I think it's fine to call loadAppVersions in the constructor rather.